Ideas-2019.md

```diff
@@ -63,6 +63,91 @@ _(in chronological order)_
 **Assisting Mentors**<br>
 None.
 
+**Work Review**<br>
+
+**Port to TelapthyGLib**<br>
+ Contributed in porting sugar, sugar-toolkit, collabwrapper.<br>
+
+Link to pull requests:<br>
+ - Sugar : https://github.com/sugarlabs/sugar/pull/837<br>
+ - Sugar-toolkit-gtk3 : https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/412<br>
+ - Collabwrapper : https://github.com/sugarlabs/collabwrapper/pull/14<br>
+
+ Other related pull requests:<br>
+ - https://github.com/sugarlabs/chat/pull/23<br>
+ - https://github.com/sugarlabs/collabedit/pull/10<br>
+ - https://github.com/sugarlabs/browse-activity/pull/97<br>
+ - https://github.com/sugarlabs/Pippy/pull/65<br>
+ - https://github.com/sugarlabs/imageviewer-activity/pull/30<br>
+ - https://github.com/sugarlabs/readetexts/pull/14<br>
+ - https://github.com/sugarlabs/sugar-docs/pull/183<br>
+
+**Testing Gwebsockets**<br>
+ Worked on testing Python 3 port of gwebsockets.<br>
+
+Link to issue:<br>
+ - https://github.com/sugarlabs/gwebsockets/issues/5<br>
+
+**Port sugar-toolkit-gtk3 to six**<br>
+   Made some fixes in toolkit version based on six and tested the toolkit.
+
+Link to issues/pull requests:<br>
+ - https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/419<br>
+ - https://github.com/sugarlabs/sugar-toolkit-gtk3/pull/420<br>
+ - https://github.com/sugarlabs/sugar-toolkit-gtk3/issues/382<br>
+
+**Port Sugar desktop to Python 3**<br>
+   Completed Python 3 port of Sugar Desktop, with the new port still demanding some fixes.<br>
+
+Link to pull request:<br>
+ - https://github.com/sugarlabs/sugar/pull/844<br>
+
+**Port Sugar datastore to Python 3**<br>
+   Completed Python 3 port of Sugar datastore.<br>
+
+Link to pull request:<br>
+ - https://github.com/sugarlabs/sugar-datastore/pull/15<br>
+
+Link to packages:<br>
+ - https://github.com/Aniket21mathur/sugar-datastore-python3
+
+**Port Sugar activities to Python 3**<br>
+   Ported sugar activities to Python 3.<br>
+
+Link to pull requests:<br>
+ - https://github.com/sugarlabs/sin-dientes-activity/pull/18<br>
+ - https://github.com/sugarlabs/nutrition/pull/7<br>
+ - https://github.com/sugarlabs/collabwrapper/pull/18<br>
+ - https://github.com/sugarlabs/browse-activity/pull/97<br>
+ - https://github.com/sugarlabs/cedit-activity/pull/15<br>
+ - https://github.com/sugarlabs/clock-activity/pull/23<br>
+
+Open pull requests:<br>
```

Comment:
PR status is dynamic and may change in future, to merged or closed. Any specific reason for adding a separate list of open PRs?